azext_edge/edge/providers/rpsaas/adr/_help.py

```diff
@@ -837,114 +837,682 @@ def load_iotops_adr_help():
+
+        examples:
+        - name: Delete an asset with confirmation prompt
+          text: >
+            az iot ops ns asset delete --name myAsset --namespace myNamespace -g myResourceGroup
+
+        - name: Delete an asset and skip the confirmation prompt
+          text: >
+            az iot ops ns asset delete --name myAsset --namespace myNamespace -g myResourceGroup -y
+    """
+
+    helps[
+        "iot ops ns asset query"
+    ] = """
+        type: command
+        short-summary: Query assets in Device Registry namespaces.
+        long-summary: |
+          Query assets across namespaces based on various search criteria including asset name,
+          device name, endpoint name and more.
+
+        examples:
+        - name: Query for a specific asset by name
+          text: >
+            az iot ops ns asset query --name myAsset -g myResourceGroup
+
+        - name: Query for assets associated with a specific device and endpoint
+          text: >
+            az iot ops ns asset query --device myDevice --endpoint-name myEndpoint -g myResourceGroup
+
+        - name: Use a custom query to search for assets
+          text: >
+            az iot ops ns asset query --custom-query "where tags.environment=='production'" -g myResourceGroup
+    """
+
+    helps[
+        "iot ops ns asset show"
+    ] = """
+        type: command
+        short-summary: Show details of an asset in a Device Registry namespace.
+
+        examples:
+        - name: Show details of an asset
+          text: >
+            az iot ops ns asset show --name myAsset --namespace myNamespace -g myResourceGroup
+    """
+
+    helps[
+        "iot ops ns asset custom"
+    ] = """
+        type: group
+        short-summary: Manage assets that point to custom device endpoints.
+    """
+
+    helps[
+        "iot ops ns asset custom create"
+    ] = """
+        type: command
+        short-summary: Create a custom asset in a Device Registry namespace.
+
+        examples:
+        - name: Create a basic custom asset
+          text: >
+            az iot ops ns asset custom create --name myCustomAsset --namespace myNamespace -g myResourceGroup
+            --device myDevice --endpoint-name myEndpoint
+
+        - name: Create a custom asset with additional metadata
+          text: >
+            az iot ops ns asset custom create --name myCustomAsset --namespace myNamespace -g myResourceGroup
+            --device myDevice --endpoint-name myEndpoint --description "Factory sensor" --display-name "Temperature Sensor"
+            --model "TempSensor-X1" --manufacturer "Contoso" --serial-number "SN12345"
+
+        - name: Create a custom asset with dataset and events configuration using inline JSON
+          text: >
+            az iot ops ns asset custom create --name myCustomAsset --namespace myNamespace -g myResourceGroup
+            --device myDevice --endpoint-name myEndpoint --dataset-config "{\\\"publishingInterval\\\": 1000}"
```

Comment:
over escaped? not sure 3 back slashes here

Reply:
you got to escape your \'s 
![image](https://github.com/user-attachments/assets/13b192ef-2d87-45fa-9d00-b6febbceda71)

the error handler for this should also have a link to our wiki page about how to properly escape in different shell environments